source/AI.cpp

```diff
@@ -1259,7 +1259,7 @@ shared_ptr<Ship> AI::FindTarget(const Ship &ship) const
 
 		// Ships which only disable never target already-disabled ships.
 		if((person.Disables() || (!person.IsNemesis() && foe != oldTarget.get()))
-				&& foe->IsDisabled() && !canPlunder)
+				&& foe->IsDisabled() && (!canPlunder || (canPlunder && Has(ship, foe->shared_from_this(), ShipEvent::BOARD))))
```

Comment:
```suggestion
				&& foe->IsDisabled() && (!canPlunder || Has(ship, foe->shared_from_this(), ShipEvent::BOARD)))
```
This might be easier to read?
(Also, I think your line length check doesn't count a tab as four characters. That ought to change.)

Reply:
> (Also, I think your line length check doesn't count a tab as four characters. That ought to change.)

I asked @quyykk when I was writing the script, and he said it should be one.